pkg/util/sys/device.go

```diff
@@ -342,11 +342,28 @@ func CheckIfDeviceAvailable(executor exec.Executor, name string, pvcBacked bool)
 			return 0, false, "", fmt.Errorf("failed to get device %s filesystem: %+v", name, err)
 		}
 	} else {
-		devFS = "" //Not checking Filesystem in case of PVC block device.
+		devFS, err = GetPVCDeviceFileSystems(executor, name)
+		if err != nil {
+			return 0, false, "", fmt.Errorf("failed to get pvc device %s filesystem: %+v", name, err)
+		}
 	}
 	return partCount, ownPartitions, devFS, nil
 }
 
+//GetPVCDeviceFileSystems returns the file system on a PVC device.
+func GetPVCDeviceFileSystems(executor exec.Executor, device string) (string, error) {
+	var fs string
+	cmd := fmt.Sprintf("get pvc filesystem type for %s", device)
+	output, err := executor.ExecuteCommandWithOutput(false, cmd, "lsblk", device, "--bytes", "--nodeps", "--noheadings", "--output", "FSTYPE")
+	if err != nil {
+		return "", fmt.Errorf("command %s failed: %+v", cmd, err)
+	}
+	logger.Infof("file system on pvc device %s - %+v", device, output)
+
+	fs = output
```

Comment:
Just return `output`, you don't need a new variable since it's not re-used anywhere. Remove `var fs string` too.